### PR TITLE
Modular files created for easier handling

### PR DESCRIPTION
**Pull request by skcy2001**
I have seperated functions or function groups of similar nature to make the code more modular and easy maintenance, I have also fixed the bug in strong_arrange.

### CODE REVIEW

**Comment by prashantsengar** [comment]
@skcy2001 can you please fix the indentation? You have perhaps set your editor to use 3 spaces for indentation, specify it to 4 please. 

And please also look at the checks made by the bot and fix the simpler ones, such as the unused imports. There are some unnecessary checks such as blank lines etc.

Can you also look at the PR for #34 ? Please add that code as well if possible

**Comment by skcy2001** [comment]
done done. I will do it.

**Comment by skcy2001** [comment]
Hey I have solved the antipattern uses and also solved issue #34

**Comment by prashantsengar** [comment]
Sorry for getting late in the review @skcy2001 
I wanted to discuss this with you whether creating so many files would be fine.
Can we merge `makeFolders`, `moveFiles`, `configure` to a single file named `utils`? And merge `strong_arrange` and `weak_arrange` both to a single `arrange`? Also, moving them to a directory called `modules` maybe so that it looks cleaner?
I want your comments on this.

2. The wildcard (*) imports are nasty for understanding. Can you fix them too, please?

**Comment by skcy2001** [comment]
Merging the two arranges are a good idea, I'll do that. About make_folders, I think it would be better if that is integrated in move_files itself and not as a seperate function. Putting them in a folder is nice too. I will do these changes.

**Comment by prashantsengar** [comment]
Task linked: [CU-1v9jca Samyak's PRs](https://app.clickup.com/t/1v9jca)

**Review by prashantsengar — APPROVED** [review]

File: lib/arrange.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+def weak_arrange(root, destination, index):
+    """
+    Deploys weak arrange function that arranges files only
+    from current folder into specified categories
+
+    """
+    TOTAL_COUNT = {}
+    for file in os.listdir(root):
+        if os.path.isfile(os.path.join(root, file)):
+            status, types = lib.utils.startProcess(root, file, index, destination)
```
Comment (prashantsengar) [inline lib/arrange.py:55]: Why are you defining `status` but not using it anywhere?

**Review by prashantsengar — COMMENTED** [review]
Also, you have named `init` as `intit.py`. Can you rename it and then check if the module works fine?

Pull this updated code as I have made a lot of changes to this PR

**Comment by prashantsengar** [comment]
You can check out https://coderwall.com/p/xyuoza/git-cloning-specific-commits if you wish to know how to clone a specific commit

File: lib/arrange.py (hunk 1 above)
Comment (skcy2001) [reply lib/arrange.py:55]: Hey! So the 'statu's was not coded by me. It was already there and I chose to not interfere with it.

**Pull request merged**